charts/dependency-track/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: 3.8.0
+description: A Helm chart for Kubernetes
+name: dependency-track
+version: 1.0.10-SNAPSHOT
```

Comment:
Il faudrait positionner une version 0.0.1 pour une première version.
Les versions SNAPSHOT ne sont pas une bonne pratique

Reply:
Il te manque différentes informations comme ce chart : https://github.com/cnieg/helm-charts/blob/master/charts/spring-boot/Chart.yaml

Reply:
Got it : https://github.com/cnieg/helm-charts/commit/2333c5d8b656120fc261d36993350f0df0bf8a60